R/utils.R

```diff
@@ -1,12 +1,14 @@
-#' Set OpenSecrets API Key	 
+#' Set OpenSecrets ENV variable OS_KEY
```

Comment:
maybe we should change it to `environment variable \code{OS_KEY}` to be extra clear